src/Sulu/Bundle/TrashBundle/Resources/config/routing_api.yaml

```
@@ -0,0 +1,31 @@
+sulu_trash.get_trash-items:
+    path: /trash-items.{_format}
+    methods: GET
+    controller: sulu_trash.trash_item_controller::cgetAction
+    format: json|csv
```

I don't know if that's an alternative syntax but in the compiled route collection the syntax is:
```suggestion
    requirements: { _format: json|csv }
```